Classes/Article.cs

```diff
@@ -0,0 +1,45 @@
+﻿
+
+namespace Classes
+{
+    public class Article
+    {
+        private int _productCode;
+        public string? _productName;

```

Comment:
Именование переменных!
```suggestion
        public string? ProductName;

```